packages/react-native-sdk/src/components/call/Lobby.tsx

```diff
@@ -21,27 +21,27 @@ import { useMediaStreamManagement } from '../../providers/MediaStreamManagement'
  * This view allows the user to toggle their audio and video state before joining a call.
  */
 /**
- * The props for the Join Button in the LobbyView.
+ * The props for the Join Button in the Lobby.
  */
 type JoinCallButton = {
   /**
-   * Handler called when the join button is clicked in the LobbyView.
+   * Handler called when the join button is clicked in the Lobby.
    * @returns void
    */
   onPressHandler: () => void;
 };
 
 /**
- * Props for the Lobby View Component
+ * Props for the Lobby Component.
  */
-type LobbyViewType = {
+type LobbyType = {
```

Comment:
`Type` suffix feels redundant. `LobbyProps` will be a better choice

Reply:
Same goes for other component props typescript as well

Reply:
Let me fix it 😄